packages/patternfly-3/patternfly-react/sass/patternfly-react/_date-time-picker.scss

```diff
@@ -0,0 +1,56 @@
+.bootstrap-datetimepicker-widget.top {
```

Comment:
this should have a -pf suffix somewhere. Probably add another class like `date-time-picker-pf` and have the `.bootstrap-datetimepicker-widget` inside of that, i.e.:
```
.date-time-picker-pf {
  &.bootstrap-datetimepicker-widget {
...
```

This will scope it to only those bootstrap date time picker components that are indeed the pf-react version. Some apps could use a mix 🤷‍♂️

Reply:
Added, thanks!